legal-api/tests/unit/resources/v2/test_business_tasks.py

```diff
@@ -296,15 +301,15 @@ def test_get_tasks_pending_correction_filings(session, client, jwt):
             LegalEntity.EntityTypes.BC_ULC_COMPANY.value,
             0,
         ),
-        ("CC first AR to be issued", "BC1234567", "2021-07-02", None, LegalEntity.EntityTypes.BC_CCC.value, 1),
-        ("CC no AR due yet", "BC1234567", "2021-07-03", None, LegalEntity.EntityTypes.BC_CCC.value, 0),
-        ("CC 3 ARs overdue", "BC1234567", "2019-05-15", None, LegalEntity.EntityTypes.BC_CCC.value, 3),
-        ("CC current AR year issued", "BC1234567", "1900-07-01", "2022-03-03", LegalEntity.EntityTypes.BC_CCC.value, 0),
-        ("CP founded in the end of the year", "CP1234567", "2021-12-31", None, LegalEntity.EntityTypes.COOP.value, 1),
-        ("CP current year AR pending", "CP1234567", "1900-07-01", "2021-03-03", LegalEntity.EntityTypes.COOP.value, 1),
-        ("CP 3 ARs overdue", "CP1234567", "2019-05-15", None, LegalEntity.EntityTypes.COOP.value, 3),
-        ("SP no AR", "FM1234567", "2019-05-15", None, LegalEntity.EntityTypes.SOLE_PROP.value, 0),
-        ("GP no AR", "FM1234567", "2019-05-15", None, LegalEntity.EntityTypes.PARTNERSHIP.value, 0),
+        ("CC first AR to be issued", "BC1234512", "2021-07-02", None, LegalEntity.EntityTypes.BC_CCC.value, 1),
+        ("CC no AR due yet", "BC1234513", "2021-07-03", None, LegalEntity.EntityTypes.BC_CCC.value, 0),
+        ("CC 3 ARs overdue", "BC1234514", "2019-05-15", None, LegalEntity.EntityTypes.BC_CCC.value, 3),
+        ("CC current AR year issued", "BC1234515", "1900-07-01", "2022-03-03", LegalEntity.EntityTypes.BC_CCC.value, 0),
+        ("CP founded in the end of the year", "CP1234561", "2021-12-31", None, LegalEntity.EntityTypes.COOP.value, 1),
+        ("CP current year AR pending", "CP1234562", "1900-07-01", "2021-03-03", LegalEntity.EntityTypes.COOP.value, 1),
+        ("CP 3 ARs overdue", "CP1234563", "2019-05-15", None, LegalEntity.EntityTypes.COOP.value, 3),
+        ("SP no AR", "FM1234561", "2019-05-15", None, LegalEntity.EntityTypes.SOLE_PROP.value, 0),
```

Comment:
Any tests with a SP will need an `alternate_name` entry to represent the business.  

You might need a `factory_alternate_name` if that doesn't exist already.  Once you have that use that to call `construct_task_list` and verify it passes the test.

Please confirm if there are any other tests like this that require a creation of a business for SPs